### DISCUSSION TITLE
100% accuracy when finetuning model

### DISCUSSION THREAD

**Discussion by nehakulkarni15** [timeseriesAI/tsai#835] [General]
Hello, I used TSAI to build a model and use the pretrained weights on a separate dataset to finetune. However, I am getting 100% accuracy when testing this dataset. Could anyone help me troubleshoot? 
![image](https://github.com/timeseriesAI/tsai/assets/114176483/a0d8f318-3e12-44b5-89a0-09b729b51faf)
![Untitled](https://github.com/timeseriesAI/tsai/assets/114176483/52bed50c-9f08-4612-89cb-b45adf99b3c4)

code: 
fineTuneName=f'{modelname}_fineTune'
dls3 = get_ts_dls(VLeftavgGut, yNew, splits=new_splits, tfms=tfms, batch_tfms=batch_tfms, path='/Users/nehakulkarni/Documents/GitHub/final-project-musculoskeletal-injury-location')
learn_pre_trained_FT = ts_learner(dls3, ResNetPlus, pretrained=True,\

                               weights_path=f'models/{modelname}_pretrain.pth',cbs = ShowGraph(),metrics=[accuracy, BalancedAccuracy(), F1Score()])
learn_pre_trained_FT.fine_tune(5, 1e-3)
learn_pre_trained_FT.export(f'{fineTuneName}.pth')
learn_pre_trained_FT.save(fineTuneName)

output: 
check unmatched_layers: ['backbone.0.convblock1.0.weight', 'backbone.0.convblock1.1.weight', 'backbone.0.convblock1.1.bias', 'backbone.0.convblock1.1.running_mean', 'backbone.0.convblock1.1.running_var', 'backbone.0.convblock2.0.weight', 'backbone.0.convblock2.1.weight', 'backbone.0.convblock2.1.bias', 'backbone.0.convblock2.1.running_mean', 'backbone.0.convblock2.1.running_var', 'backbone.0.convblock3.0.weight', 'backbone.0.convblock3.1.weight', 'backbone.0.convblock3.1.bias', 'backbone.0.convblock3.1.running_mean', 'backbone.0.convblock3.1.running_var', 'backbone.0.shortcut.0.weight', 'backbone.0.shortcut.1.weight', 'backbone.0.shortcut.1.bias', 'backbone.0.shortcut.1.running_mean', 'backbone.0.shortcut.1.running_var', 'backbone.1.convblock1.0.weight', 'backbone.1.convblock1.1.weight', 'backbone.1.convblock1.1.bias', 'backbone.1.convblock1.1.running_mean', 'backbone.1.convblock1.1.running_var', 'backbone.1.convblock2.0.weight', 'backbone.1.convblock2.1.weight', 'backbone.1.convblock2.1.bias', 'backbone.1.convblock2.1.running_mean', 'backbone.1.convblock2.1.running_var', 'backbone.1.convblock3.0.weight', 'backbone.1.convblock3.1.weight', 'backbone.1.convblock3.1.bias', 'backbone.1.convblock3.1.running_mean', 'backbone.1.convblock3.1.running_var', 'backbone.1.shortcut.0.weight', 'backbone.1.shortcut.1.weight', 'backbone.1.shortcut.1.bias', 'backbone.1.shortcut.1.running_mean', 'backbone.1.shortcut.1.running_var', 'backbone.2.convblock1.0.weight', 'backbone.2.convblock1.1.weight', 'backbone.2.convblock1.1.bias', 'backbone.2.convblock1.1.running_mean', 'backbone.2.convblock1.1.running_var', 'backbone.2.convblock2.0.weight', 'backbone.2.convblock2.1.weight', 'backbone.2.convblock2.1.bias', 'backbone.2.convblock2.1.running_mean', 'backbone.2.convblock2.1.running_var', 'backbone.2.convblock3.0.weight', 'backbone.2.convblock3.1.weight', 'backbone.2.convblock3.1.bias', 'backbone.2.convblock3.1.running_mean', 'backbone.2.convblock3.1.running_var', 'backbone.2.shortcut.weight', 'backbone.2.shortcut.bias', 'backbone.2.shortcut.running_mean', 'backbone.2.shortcut.running_var']

**Comment by nehakulkarni15** [comment]
Also, when I try to display the confusion matrix I get this
![image](https://github.com/timeseriesAI/tsai/assets/114176483/ec46c083-4b65-4e2d-9806-676a1d758f29)
